lib/licensify.js

```diff
@@ -97,15 +97,55 @@ function createLicenseHeader (licenses) {
 
 module.exports = function licensify (b, opts) {
     var licenses = {};
+    var checkedPkg = [];
 
     b.on('package', function (pkg) {
-        var summary = {};
-        extractLicense(pkg, summary);
-        extractLicenses(pkg, summary);
-        extractAuthor(pkg, summary);
-        extractMaintainers(pkg, summary);
-        extractContributors(pkg, summary);
-        licenses[pkg.name] = summary;
+        var extract = function(pkg) {
+            var summary = {};
+            extractLicense(pkg, summary);
+            extractLicenses(pkg, summary);
+            extractAuthor(pkg, summary);
+            extractMaintainers(pkg, summary);
+            extractContributors(pkg, summary);
+            return summary;
+        };
```

Comment:
Please move `extract` function out of the `module.exports`. In other word, please put `extract` function on the same level with `extractLicense`, `extractLicenses` and so on.